code/parachain/runtime/composable/src/lib.rs

```diff
@@ -667,6 +707,9 @@ mod benches {
 		[proxy, Proxy]
 		[assets_registry, AssetsRegistry]
 		[multisig, Multisig]
+		[whitelist, Whitelist]
```

Comment:
tbh let not add any benches now. let use parity weights. we can think of weigts later.